### DISCUSSION TITLE
Need Help

### DISCUSSION THREAD

**Discussion by pgalinta** [globaleaks/globaleaks-whistleblowing-software#4244] [General]
### Proposal

We need to integrate within our corporate dedicated page a privacy and cookie policy.

### Motivation and context

Suggested by our privacy gurantor

**Comment by evilaliv3** [comment]
Thank you for you question @pgalinta 

The platform enable to configure a privacy policy url  in the footer that you can point to an external site or to a PDF loaded as a file on the platform.
You could do this in the Admin/Content settings.

If you, instead, need some professional support you could write your needs to support@globaleaks.org with your specific needs and we may try to see if we could assist you via our social enterprise specifically incorporated to sustain the globaleaks research and developmet by providing professional support.
